crates/rpc/rpc-eth-api/src/helpers/fee.rs

```diff
@@ -154,11 +155,23 @@ pub trait EthFees: LoadFee {
                 base_fee_per_blob_gas.push(last_entry.next_block_blob_fee().unwrap_or_default());
             } else {
```

Comment:
there are two modes in this impl, one where everything is cached,, the if branch of this else branch, in which we'd also need to add an additional entry for the pending block.

so it would make sense to calculate the pending things above the if statement and then update the fee entries or the non-cached mode accordingly